docs: add documentation for 802.1x Wifi (#4896)
eclipse/kura: docs/gateway-configuration/wifi-configuration-8021x.md

# Wi-Fi 802.1x Configuration

Enterprise Wi-Fi is currently only supported on gateways running our [generic profiles](/getting-started/install-kura/#installer-types). The following is a list of currently supported 802.1x configurations.

 - TTLS-MSCHAPv2
 - PEAP-MSCHAPv2

>  - TLS-EAP

Reviewer: I was kinda expecting to find 3 section, one for each of the authentication methods. I have two instead, one of which combines TTLS and PEAP. It's fine but I would add a short sentence explaining that, since they are so similar, the documentation will explain how to use both of them at the same time.... or something along those lines.

> # Setup TTLS/PEAP-MSCHAPv2

Reviewer: H2 title

```suggestion
## Setup TTLS/PEAP-MSCHAPv2
```

 1. Set up gateway Wi-Fi as described in the [Wi-Fi configuration guide](/gateway-configuration/wifi-configuration/#wireless-configuration).
 2. Ensure Wireless Security is set to `WPA2/WPA3-Enterprise`
    ![Alt text](images/802-1x-images/wifi-enterprise-enum.png)
 3. select the 802.1x tab
 4. Set Enteprise EAP -> `TTLS/PEAP`
 5. Set Inner Authentication -> `MSCHAPV2`
 6. Set `Identity (Username)` 
 7. Set `Password`
 6. Press 'Apply'

The configuration should look like the following:
![Alt text](images/802-1x-images/wifi-ttls.png)

> # Setup TLS-EAP

Reviewer: H2 title

```suggestion
## Setup TLS-EAP
```

To connect via TLS-EAP you will need the following items in unencrypted PEM format:
    - Certificate Authority (CA) Certificate

>     - Client Certificate + Private Key (PKCS8)

Reviewer: Not rendering correctly

![image](https://github.com/eclipse/kura/assets/22748355/d0b07c34-9456-4181-8787-31292f5c0994)

Should be:

```
# Setup TLS-EAP
To connect via TLS-EAP you will need the following items in unencrypted PEM format:

- Certificate Authority (CA) Certificate
- Client Certificate + Private Key (PKCS8)
```

> Enrolling secrets in the Keystore service

Reviewer: I would add a link to the existing documentation: https://eclipse.github.io/kura/docs-release-5.3/gateway-configuration/keystores-management/

> Enrolling secrets in the Keystore service

Reviewer: This could also be a H3 title?

```suggestion
### Enrolling secrets in the Keystore service
```

 1. Navigate to `Security` under the `System` tab.
 2. Under the `Keystore Configuration` add a new keystore, and keep note of the name. ![Adding a new keystore](images/802-1x-images/wifi-create-keystore.png)
 3. After the Keystore is created, be sure to change the path to a persistent directory. ![Alt text](images/802-1x-images/wifi-create-change-path.png)
 4. Navigate to the `Certificate List` and create a new Certificate. Insert the PEM and Apply, keep note of the name. ![add certificate](images/802-1x-images/wifi-create-keystore-add-certificate.png)
 5. Now press `add` and create a new Private Key. Insert both the certificates in the PEM in the dialogue and press apply. keep note of the name. ![Alt text](images/802-1x-images/wifi-create-keystore-add-Keystore.png)

> Wifi Setup

Reviewer: This could also be a H3 title?

```suggestion
### Wifi Setup
```

 1. Set up gateway Wi-Fi as described in the [Wi-Fi configuration guide](/gateway-configuration/wifi-configuration/#wireless-configuration).
 2. Ensure Wireless Security is set to `WPA2/WPA3-Enterprise`. ![Alt text](images/802-1x-images/wifi-enterprise-enum.png)
 3. Select the 802.1x tab.
 4. Set Enteprise EAP -> `TLS`.
 5. Set Inner Authentication -> `MSCHAPV2`.

>  6. Set `Identity (Username)`.

Reviewer: ... is this really needed? I've not changed it in my tests and it seems to have no effect on the connection...

# Wi-Fi 802.1x Configuration

> Enterprise Wi-Fi is currently only supported on gateways running our [generic profiles](/getting-started/install-kura/#installer-types). The following is a list of currently supported 802.1x configurations.

Reviewer: ```suggestion
Enterprise Wi-Fi is currently only supported on gateways running our [generic profiles](/getting-started/install-kura/#installer-types). The following is a list of currently supported 802.1x authentication methods.
```
